requirements.txt

```diff
@@ -1,12 +1,120 @@
-dash>=1.11,<1.21.0
-dash-core-components
-dash-html-components
-dash-renderer
-plotly
-dpd-components
-
-dash-bootstrap-components
-
-channels<3.0
-Django>=2.2
-Flask>=1.0.2
+aioredis==1.3.1
```

Comment:
This file needs to be version-independent as much as possible - otherwise it will force users into using specific versions.
Ideally, the only version constraints will be for compatibility reasons, such as a minimum Django version, or for avoiding backwards-incompatible changes, such as the current channels version constraint.